src/main/java/Model/Web/Project.java

```diff
@@ -0,0 +1,49 @@
+package Model.Web;
+
+import com.google.gson.annotations.Expose;
+
+import java.sql.Date;
+
+public class Project extends PrettyObject{
+    // PARAMETERS
+    @Expose
+    private int id;
```

Comment:
Akoze ja v serializovatelnych modeloch radsej pouzivam Integer, ale je to skor len osobna skusenost ktora predchadza roznym problemom neskor.

Reply:
vies povedat nejaku ukazku tych problemov?